src/bignum.rs

```diff
@@ -0,0 +1,39 @@
+//! The bignum system library.
+
+use super::*;
+
+mod native {
```

Comment:
Don't we want to expose the unsafe native methods? maybe for use-cases where shaving a bit of overhead matters

Reply:
True, good idea.

Reply:
I'm not fully convinced that there is much overhead in the wrapper fn, but this should be fine. Just need to leave a note to avoid using the native API.

Reply:
Added.